cosmicds/app.vue

```diff
@@ -53,7 +53,7 @@
                 v-on="on"
                 @click="speech_menu = !speech_menu"
               >
-                <v-icon>mdi-voice</v-icon>
+                <v-icon>mdi-tune</v-icon>
```

Comment:
Perhaps use [`mdi-tune-vertical`](https://pictogrammers.com/library/mdi/icon/tune-vertical/). Both here and L292